torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -2634,7 +2634,7 @@ struct to_ir {
         return use_inplace_op ? aten::bitwise_xor : aten::__xor__;
       case TK_LSHIFT:
         // NOLINTNEXTLINE(bugprone-branch-clone)
```

Comment:
```suggestion
```
Oh wow, Clang-Tidy caught this, but we ignored it with a comment. Please remove the clang-tidy comment.